Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md

```diff
@@ -20,10 +12,14 @@ There are two application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow us to access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).
+To use the **Cortex XSOAR application** and allow Cortex XSOAR access to Microsoft Graph Security an administrator has to approve our app using an admin consent flow by clicking this **[link](https://oproxy.demisto.ninja/ms-graph-security)**.
```

Comment:
Recommend:
To allow Cortex XSOAR access to Microsoft Graph Security, an administrator has to approve the Cortex XSOAR app using an admin consent flow by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).